snowshu/core/docker.py

```diff
@@ -82,6 +82,7 @@ def convert_container_to_replica(
     def get_stopped_container(  # noqa pylint: disable=too-many-arguments
```

Comment:
If we're going to be referencing and working with the expected image for the target adapter, we should start passing in the target adapter as an argument instead of directly referencing the image via `POSTGRES_IMAGE`.

The functionality here in `docker.py` should essentially be image agnostic with the exception of some of the architecture verification.

Reply:
Done